DEPENDS.txt

```diff
@@ -54,9 +54,9 @@ functionality is only available with the following installed:
 
 Testing requirements
 --------------------
-* `Nose <https://nose.readthedocs.org/en/latest/>`__
+* `pytest <http://doc.pytest.org/en/latest/>`__
       A Python Unit Testing Framework
-* `Coverage.py <http://nedbatchelder.com/code/coverage/>`__
+* `pytest-cov <http://pytest-cov.readthedocs.io/en/latest/>`__
```

Comment:
This will require updating the guidelines in http://scikit-image.org/docs/dev/contribute.html#test-coverage .